CONTRIBUTING.md

```diff
@@ -108,23 +108,18 @@ helm install consul --create-namespace -n consul -f ./values.dev.yaml ./charts/c
 
 ### Building and running the `consul-k8s` CLI
 
-Change directory into the `cli` folder where the golang code resides.
+Compile the `consul-k8s` CLI binary for your local machine:
 
 ```shell
-cd cli
+make cli-dev
 ```
+This will compile the `consul-k8s` binary into `cli/bin/consul-k8s` as well as your `$GOPATH`.
 
-Build the CLI binary using the following command
+Run the CLI as follows:
 
 ```shell
-go build -o bin/consul-k8s
-```
-
-Run the CLI as follows
-
-```shell
-./bin/consul-k8s version
-consul-k8s 0.36.0-dev
+consul-k8s version
+consul-k8s v0.48.0-dev
```

Comment:
I think you can cut this second line. It adds overhead that we have to update and should just be the response from the `version` subcommand.

Reply:
Sounds good, yeah, I thought it looked a bit weird. I'll remove it